_includes/guide_transactions.md

```diff
@@ -211,38 +211,37 @@ of the process.
   Because it's just data, nothing is done except adding it to the stack.
   The public key (also from the signature script) is pushed on top of the signature.
 
-* From Alice's pubkey script, the `OP_DUP` operation is pushed. `OP_DUP` replaces
-  itself with a copy of the data from one level below it---in this
+* From Alice's pubkey script, the `OP_DUP` operation is executed. `OP_DUP` pushes onto the stack
+  a copy of the data currently at the top of it---in this
   case creating a copy of the public key Bob provided.
 
-* The operation pushed next, `OP_HASH160`, replaces itself with a hash
-  of the data from one level below it---in this case, Bob's public key.
+* The operation executed next, `OP_HASH160`, pushes onto the stack a hash
+  of the data currently on top of it---in this case, Bob's public key.
   This creates a hash of Bob's public key.
 
 * Alice's pubkey script then pushes the pubkey hash that Bob gave her for the
   first transaction.  At this point, there should be two copies of Bob's
   pubkey hash at the top of the stack.
 
-* Now it gets interesting: Alice's pubkey script adds `OP_EQUALVERIFY` to the
-  stack. `OP_EQUALVERIFY` expands to `OP_EQUAL` and `OP_VERIFY` (not shown).
+* Now it gets interesting: Alice's pubkey script executes `OP_EQUALVERIFY`.
+  `OP_EQUALVERIFY` is like executing `OP_EQUAL` followed by `OP_VERIFY` (not shown).
 
     `OP_EQUAL` (not shown) checks the two values below it; in this
```

Comment:
"two values below it" doesn't make sense now that we're not talk as though opcodes are on the stack; this should be changed to "two values at the top of the stack".